### PR TITLE
Remove unnecessary sudo calls

### PR DESCRIPTION
**Pull request by lpiwowar**

### CODE REVIEW

**Comment by lpiwowar** [comment]
The job is failing on (probably not issue caused by the PR):
```
2024-09-02 07:42:11.513246 | controller | TASK [edpm_deploy_baremetal : Wait for OpenStackDataPlaneNodeSet to be deployed _raw_params=oc wait OpenStackDataPlaneNodeSet {{ cr_name }} --namespace={{ cifmw_install_yamls_defaults['NAMESPACE'] }} --for=condition=ready --timeout={{ cifmw_edpm_deploy_baremetal_wait_dataplane_timeout_mins }}m] ***
2024-09-02 07:42:11.513256 | controller | Monday 02 September 2024  07:02:11 -0400 (0:00:00.050)       0:28:27.974 ******
2024-09-02 07:42:11.513277 | controller | fatal: [localhost]: FAILED! => changed=true
2024-09-02 07:42:11.636011 | controller |   cmd:
2024-09-02 07:42:11.636102 | controller |   - oc
2024-09-02 07:42:11.636111 | controller |   - wait
2024-09-02 07:42:11.636117 | controller |   - OpenStackDataPlaneNodeSet
2024-09-02 07:42:11.636125 | controller |   - openstack-edpm-ipam
2024-09-02 07:42:11.636131 | controller |   - --namespace=openstack
2024-09-02 07:42:11.636138 | controller |   - --for=condition=ready
2024-09-02 07:42:11.636144 | controller |   - --timeout=40m
2024-09-02 07:42:11.636150 | controller |   delta: '0:40:00.143228'
2024-09-02 07:42:11.636156 | controller |   end: '2024-09-02 07:42:11.487006'
2024-09-02 07:42:11.636163 | controller |   msg: non-zero return code
2024-09-02 07:42:11.636169 | controller |   rc: 1
2024-09-02 07:42:11.636290 | controller |   start: '2024-09-02 07:02:11.343778'
2024-09-02 07:42:11.636301 | controller |   stderr: 'error: timed out waiting for the condition on openstackdataplanenodesets/openstack-edpm-ipam'
2024-09-02 07:42:11.636309 | controller |   stderr_lines: <omitted>
2024-09-02 07:42:11.636315 | controller |   stdout: ''
2024-09-02 07:42:11.636321 | controller |   stdout_lines: <omitted>
```

**Comment by lpiwowar** [comment]
recheck

**Review by kopecmartin — APPROVED** [review]
lgtm as long as it works

**Review by ashu-011 — APPROVED** [review]

File: container-images/tcib/base/os/horizontest/run_horizontest.sh [hunk 1]
```diff
@@ -102,7 +102,7 @@ function delete_custom_resources {
 
 pushd ${HORIZONTEST_DIR}
 git clone ${REPO_URL} ${HORIZONTEST_DIR}/horizon
-sudo chown -R horizontest:horizontest horizon
+chown -R horizontest:horizontest horizon
```
Comment (son-vyas) [inline container-images/tcib/base/os/horizontest/run_horizontest.sh:105]: chown will require sudo privilege

File: container-images/tcib/base/tobiko/run_tobiko.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ TOBIKO_LOGS_DIR_NAME=${TOBIKO_LOGS_DIR_NAME:-"tobiko"}
 [ ! -z ${TOBIKO_NUM_PROCESSES} ] && export TOX_NUM_PROCESSES=${TOBIKO_NUM_PROCESSES}
 
 pushd ${TOBIKO_DIR}
-sudo chown tobiko:tobiko -R tobiko
+chown tobiko:tobiko -R tobiko
```
Comment (son-vyas) [inline container-images/tcib/base/tobiko/run_tobiko.sh:37]: same here?

**Pull request merged**